.github/workflows/ci.yml

```diff
@@ -3,10 +3,16 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  build:
 
+  prepare:
     runs-on: ubuntu-latest
+    if: "! contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
Yes, but it does not work well :( I think, I used up some of you free GH-Workflow minutes. I hope they come from my fork's contingent instead of draining your minutes-allowance for this month.

Reply:
My understanding is, that GH Actions are free for open source - only private repos are limited with (free) minutes:

https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-billing-for-github-actions

```
GitHub Actions usage is free for public repositories. For private repositories, each GitHub account receives a certain amount of free minutes and storage ..
```

In the settings I can also confirm that no free minutes have been used. 
Anyways, a `skip` functionality doesn't harm :+1: